src/pipe.c

```diff
@@ -62,6 +62,31 @@ static int luv_pipe_connect(lua_State* L) {
   return 1;
 }
 
+
+#if LUV_UV_VERSION_GEQ(1,46,0)
+static int luv_pipe_bind2(lua_State* L) {
+  size_t namelen;
+  uv_pipe_t* handle = luv_check_pipe(L, 1);
+  const char* name = luaL_checklstring(L, 2, &namelen);
+  unsigned int flags = luaL_optinteger(L, 3, 0);
```

Comment:
Would be good to allow this (and flags in `connect2`) to be an integer, table, or nil. Similar to this:

https://github.com/luvit/luv/blob/71d6dd7772f6a70effeb9e6fe5cf221f44cf88a9/src/misc.c#L709-L720

and the `TTABLE` handled like this:

https://github.com/luvit/luv/blob/71d6dd7772f6a70effeb9e6fe5cf221f44cf88a9/src/udp.c#L144-L152

(the field to check should probably be named `"no_truncate"`)

Reply:
resolved, introduce new `luv_pipe_optflags`